src/+otp/+brusselator/BrusselatorProblem.m

```diff
@@ -1,34 +1,55 @@
 classdef BrusselatorProblem < otp.Problem
-    % BrusselatorProblem A two-variable model for a system of autocatalytic
-    %   reaction equations
+    %BRUSSELATORPROBLEM A two-variable model for an autocatalytic reaction
+    %   The Brusselator chemical reaction is given by
     %
     %        A → X
     %    B + X → Y + D
     %   2X + Y → 3X
     %        X → E
     %
-    %   This system can be reduced to the following two equations
+    %   With the assumption that all reaction rates are one and the concentrations of A and B are constant parameters,
```

Comment:
I am not sure that this is true either. we are simply reducing the number of reactions to two and non-dimensionalizing the problem.